test_runlike.py

```diff
@@ -152,3 +152,9 @@ def test_cap_add(self):
     def test_devices(self):
         self.expect_substr("--device /dev/null:/dev/null:r")
         self.expect_substr("--device /dev/null:/dev/null", 2)
+
+    def test_entrypoint(self):
+        self.expect_substr("--entrypoint /bin/bash", 6)
```

Comment:
I think it's a good idea to test that `--entrypoint` does not get printed when the inspected container does not have it specified. e.g. in fixture 1, expect not to see `--entrypoint`